docs/contributing/intro.md

```diff
@@ -4,20 +4,12 @@ sidebar_position: 1
 
 # Getting Started
 
-The best way to get connected with the Arrow community and start working is to join our Discord server. Introduce yourself in the "introductions" channel and we'll all be happy to get you up to speed - we're a friendly bunch!
+The best way to get connected with the Arrow community and start working is to join our Discord server!
 
-As a general rule of culture, you don't need formal permission to start working on anything, but it's best to communicate well with the community to keep us all aligned and working on high-impact tasks. 
+Introduce yourself in the `#introductions` channel and we'll all be happy to get you up to speed - we're a friendly bunch!
 
-As of now, we're still finalizing the details of our Arrow token, but we will make sure that all contributors are retroactively rewarded for all of their hard work. We strive to have a culture of merit - If you contribute something valuable, you will be rewarded.
+:warning: As a general rule of culture, you don't need formal permission to start working on anything. However, it's best to communicate well with the community to keep us all aligned and working on high-impact tasks.
```

Comment:
This doesn't render well. I think the emoji intended is the yellow triangle with exclamation mark? That emoji sometimes only renders as a small black-and-white symbol. 
<img width="253" alt="Screen Shot 2022-07-08 at 11 31 00 AM" src="https://user-images.githubusercontent.com/23110002/178050207-51b082c0-df72-41b8-9b98-74dfeca7a187.png">

Suggest to replace with ❗️

Reply:
Fixed in d84a92a